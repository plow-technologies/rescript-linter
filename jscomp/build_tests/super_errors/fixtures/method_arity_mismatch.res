let f = obj => {
  obj["hi"](.1, 2)
  obj["hi"](.1)
}
