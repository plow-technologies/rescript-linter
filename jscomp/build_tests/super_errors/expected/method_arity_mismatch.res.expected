
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/method_arity_mismatch.res[0m:[2m3:3-11[0m

  1 [2mâ”‚[0m let f = obj => {
  2 [2mâ”‚[0m   obj["hi"](.1, 2)
  [1;31m3[0m [2mâ”‚[0m   [1;31mobj["hi"][0m(.1)
  4 [2mâ”‚[0m }
  5 [2mâ”‚[0m 

  This uncurried function has type (. int, int) => unit
  It is applied with [1;31m1[0m arguments but it requires [1;33m2[0m.